src/test/script_standard_tests.cpp

```diff
@@ -197,14 +197,22 @@ BOOST_AUTO_TEST_CASE(script_standard_Solver_failure)
     s << OP_RETURN << std::vector<unsigned char>({75}) << OP_ADD;
     BOOST_CHECK_EQUAL(Solver(s, solutions), TxoutType::NONSTANDARD);
 
-    // TxoutType::WITNESS_UNKNOWN with incorrect program size
+    // TxoutType::WITNESS_V0_{KEY,SCRIPT}HASH with incorrect program size (-> consensus-invalid, i.e. non-standard)
```

Comment:
Should this be:
```suggestion
    // TxoutType::WITNESS_V0_{KEY,SCRIPT}HASH with incorrect program size (-> policy-invalid, i.e. non-standard)
```
?

Seems like the terminology could be out of sync with the comment below (mentioning "policy" instead of "consensus"). Or is it that segwit v0 program sizes are stricter and v1 program sizes are more flexible, consensus-wise?

Reply:
Thanks for the references! I'm still unfamiliar with that part. So for `MANDATORY_SCRIPT_VERIFY_FLAGS` we reach the return true there for Taproot. :+1: